domain/domain.go

```diff
@@ -1294,6 +1389,10 @@ func (do *Domain) runawayRecordFlushLoop() {
 			// flush as soon as possible.
 			if len(quarantineRecordCh) == 0 || len(quarantineRecords) >= flushThrehold {
 				flushQuarantineRecords()
+				if owner.IsOwner() {
```

Comment:
Why need gc here, don't get the point

Reply:
Because when the watch is updated, it may be a new identify, at this time, I think I can delete the old one immediately.